website/package.json

```diff
@@ -15,7 +15,7 @@
     "@hashicorp/react-head": "1.1.6",
     "@hashicorp/react-image": "3.0.3",
     "@hashicorp/react-product-downloader": "7.0.1",
-    "@hashicorp/react-section-header": "3.0.1",
+    "@hashicorp/react-section-header": "4.0.0",
```

Comment:
The only direct use of `section-header` is on the `community` page, and the content passed does not use markdown, so no other changes were needed